README.rst

```diff
@@ -18,45 +31,48 @@ or, if you know you want to create a development Account, you can omit that argu
 
 There are two constants available to you to specify the url for either the development environment
 or the production environment, which are:
-- `fxapom.DEV_URL` - the url for the development environment
-- `fxapom.PROD_URL` - the url for the production environment
+* `fxapom.DEV_URL` - the url for the development environment
```

Comment:
This is not rendering correctly in GitHub, I suspect we need a blank line before the list?